x-pack/plugins/security_solution/public/resolver/test_utilities/connect_enzyme_wrapper_and_store.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Store } from 'redux';
+import { ReactWrapper } from 'enzyme';
```

Comment:
:grey_question: link to enzyme docs?

Reply:
what exactly are you looking for? 

this is about the `mount` function which returns `ReactWrapper` https://enzymejs.github.io/enzyme/docs/api/mount.html

This is the docs for the `update` function. https://enzymejs.github.io/enzyme/docs/api/ReactWrapper/update.html
Those might be more useful since this file is really about automatically calling that.